MFaaP.MFWSClient/MFWSVaultObjectSearchOperations.cs

```diff
@@ -27,10 +27,11 @@ internal MFWSVaultObjectSearchOperations(MFWSClientBase client)
 		/// </summary>
 		/// <param name="searchTerm">The string to search for.</param>
 		/// <param name="objectTypeId">If provided, also restricts the results by the given object type.</param>
+		/// <param name="limit">If 0, uses Mfiles default at server (default 500 items returned)</param>
 		/// <param name="token">A cancellation token for the request.</param>
 		/// <returns>An array of items that match the search term.</returns>
 		/// <remarks>For more comprehensive search options, construct a series of <see cref="ISearchCondition"/> objects and use the <see cref="SearchForObjectsByString"/> method.</remarks>
-		public Task<ObjectVersion[]> SearchForObjectsByStringAsync(string searchTerm, int? objectTypeId = null, CancellationToken token = default(CancellationToken))
+		public Task<ObjectVersion[]> SearchForObjectsByStringAsync(string searchTerm, int? objectTypeId = null, int limit = 0, CancellationToken token = default(CancellationToken))
```

Comment:
Use a constant rather than the hard-coded number.

Reply:
solution in new PR